app/helpers/meta_tags_helper.rb

```diff
@@ -0,0 +1,25 @@
+  # frozen_string_literal: true
+
+  # Helper for meta tags to assign default tags or page-specific content_tag
+
+  def meta_title
+    content_for?(:meta_title) ? content_for(:meta_title) : DEFAULT_META['meta_title']
```

Comment:
I think this can also be expressed as:
`content_for?(:meta_title) || DEFAULT_META['meta_title']`

Reply:
Just saw these. Will implement

Reply:
this is done.